give.php

```diff
@@ -524,5 +525,13 @@ function give($abstract = null)
 }
 
 require __DIR__ . '/vendor/autoload.php';
+require __DIR__ . '/vendor/vendor-prefixed/autoload.php';
+
+// Setup and initialize the Validation library
+ValidationConfig::setServiceContainer(give());
+ValidationConfig::setHookPrefix('givewp_');
+ValidationConfig::setInvalidArgumentExceptionClass(InvalidArgumentException::class);
+ValidationConfig::setValidationExceptionClass(ValidationException::class);
+ValidationConfig::initialize();
```

Comment:
Can we please move this into its own service provider?

Reply:
Hah! Yeah, I kept debating that. Thanks for pushing me in that direction. Resolved in eef378d9a